odoo/tests/common.py

```diff
@@ -445,7 +445,7 @@ def phantom_run(self, cmd, timeout):
             _logger.info('phantomjs unlink localstorage %s', i)
             os.unlink(i)
         try:
-            phantom = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=None, close_fds=True)
+            phantom = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=None)
```

Comment:
no idea why your branch should change this. if this is necessary, should be in another commit with a good description of the issue in the commit message